src/main.rs

```diff
@@ -2,23 +2,41 @@ use zellij_tile::prelude::*;
 
 use std::collections::BTreeMap;
 
-#[derive(Default)]
-struct State {}
+struct State {
+    confirm_key: Key,
+    abort_key: Key,
```

Comment:
nit: can you change `abort_key` to `cancel_key`?  help text uses `Cancel`:
```
Help: <ENTER> - Confirm, <Esc> - Cancel
```

Reply:
Sure, I renamed it